tsconfig.json

```diff
@@ -15,9 +15,13 @@
     "jsxImportSource": "@emotion/react",
     "incremental": true,
     "downlevelIteration": true,
-    "types": ["cypress"],
+    "types": ["cypress", "@testing-library/cypress"],
     "isolatedModules": true
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "next.config.js"],
-  "exclude": ["node_modules", "src/@types/resolvers-types.ts"]
+  "exclude": [
+    "node_modules",
+    "src/@types/resolvers-types.ts",
+    "scripts/areaToDatabase/*"
```

Comment:
`areaToDatabase` is an archive of old js code which we will eventually re-write so we have the ability to populate our areas table from scratch again

Reply:
so we exclude it for now as it won't pass the type checker